src/intl/en/page-upgrades-get-involved-bug-bounty.json

```diff
@@ -1,32 +1,37 @@
-  "page-upgrades-bug-bounty-hunting-li-1": "Issues that have already been submitted by another user or are already known to spec and client maintainers are not eligible for bounty rewards.",
+  "page-upgrades-bug-bounty-hunting-execution-leaderboard-subtitle": "Find execution layer bugs to get added to this leaderboard",
+  "page-upgrades-bug-bounty-hunting-li-1": "Issues without a POC or that have already been submitted by another user or are already known to spec and client maintainers are not eligible for bounty rewards.",
   "page-upgrades-bug-bounty-hunting-li-2": "Public disclosure of a vulnerability makes it ineligible for a bounty.",
   "page-upgrades-bug-bounty-hunting-li-3": "Ethereum Foundation researchers and employees of consensus layer client teams are not eligible for rewards.",
   "page-upgrades-bug-bounty-hunting-li-4": "Ethereum bounty program considers a number of variables in determining rewards. Determinations of eligibility, score and all terms related to an award are at the sole and final discretion of the Ethereum Foundation bug bounty panel.",
   "page-upgrades-bug-bounty-leaderboard": "See full leaderboard",
   "page-upgrades-bug-bounty-leaderboard-points": "points",
-  "page-upgrades-bug-bounty-ledger-desc": "The Beacon Chain specification details the design rationale and proposed changes to Ethereum via the Beacon Chain upgrade.",
-  "page-upgrades-bug-bounty-ledger-title": "The Beacon Chain specification bugs",
+  "page-upgrades-bug-bounty-ledger-desc": "The Ethereum Specifications details the design rationale for the Execution Layer and Consensus Layer.",
+  "page-upgrades-bug-bounty-ledger-title": "Specification bugs",
   "page-upgrades-bug-bounty-meta-description": "An overview of the consensus layer bug bounty program: how to get involved and reward information.",
-  "page-upgrades-bug-bounty-meta-title": "Consensus layer bug hunting bounty program",
-  "page-upgrades-bug-bounty-not-included": "Not included",
-  "page-upgrades-bug-bounty-not-included-desc": "The Merge and shard chain upgrades are still in active development and so are not yet included as part of this bounty program.",
+  "page-upgrades-bug-bounty-meta-title": "Bug Bounty Program",
+  "page-upgrades-bug-bounty-not-included": "Out of scope",
+  "page-upgrades-bug-bounty-not-included-desc": "Only the targets listed under in-scope are part of the Bug Bounty Program. This means that for example our infrastructure; such as webpages, dns, email etc, are not part of the bounty-scope. The Merge and shard chain upgrades are still in active development and so are not yet included as part of this bounty program. ERC20 contract bugs are typically not included in the bounty scope. However, we can help reach out to affected parties, such as authors or exchanges in such cases. ENS is maintained by the ENS foundation, and is not part of the bounty scope.",
```

Comment:
We should make sure the deposit  contract is explicitly in scope. It is part of the consensus specs but might be worth calling out directly